truffle.js

```diff
@@ -4,4 +4,11 @@ require('babel-polyfill');
 module.exports = {
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
+  networks: {
+    test: {
```

Comment:
I don't think this is necessary-- doesn't `truffle test` already default to this?

Reply:
true. it is supposed to default to the test-rpc baked into truffle. But, when I ran it, it couldn't find a test network to connect to. I think ill upgrade the truffle and solidity versions.